docs/en/observability/cloud-monitoring/aws/monitor-amazon-intro.asciidoc

```diff
@@ -1,3 +1,5 @@
+:aws: AWS
```

Comment:
If you want to use this attribute, you'll need to set it at the global level ([here](https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc)) or at each page level. We no longer support book-scoped attributes as of Mike's PR a few days ago.

My suggestion would be to remove this and update all of the `{aws}` uses to `AWS`.

Reply:
@bmorelli25 
I opened this [PR](https://github.com/elastic/docs/pull/2973) to set the `:aws: AWS` attribute at the global level. I guess I can remove the book-scoped attributes after the #2973 is merged. Can you confirm?

Reply:
That's correct! The docs PR LGTM

Reply:
```suggestion
```